llvm/include/llvm/CodeGen/ValueTypes.td

```diff
@@ -273,20 +282,54 @@ def nxv2f64  : VTScalableVec<2,  f64, 187>;  // n x  2 x  f64 vector value
 def nxv4f64  : VTScalableVec<4,  f64, 188>;  // n x  4 x  f64 vector value
 def nxv8f64  : VTScalableVec<8,  f64, 189>;  // n x  8 x  f64 vector value
 
-def x86mmx    : ValueType<64,   190>;  // X86 MMX value
-def Glue      : ValueType<0,    191>;  // Pre-RA sched glue
-def isVoid    : ValueType<0,    192>;  // Produces no value
-def untyped   : ValueType<8,    193> { // Produces an untyped value
+// Sz = NF * LMUL * BitsPerBlock
+def riscv_mf8x2 : VTVecTup<16, 2, i8, 190>;  // RISCV vector tuple(lmul=mf8, nf=2)
```

Comment:
I think we should avoid putting lmul in the name. We will need to support two different values for RVVBitsPerBlock someday. We need to be able to identify the types based on `nf x min_elts x i8`. I think the alternate RVVBitsPerBlock=32 list of types is a subset of RVVBitsPerBlock=64.

Reply:
That's a good point that I didn't think about, but one question, if we support more than one `RVVBitsPerBlock`, should it be equal to `ELEN`?

Reply:
> That's a good point that I didn't think about, but one question, if we support more than one `RVVBitsPerBlock`, should it be equal to `ELEN`?

Yes it should be the same as ELEN. The main reason we haven't done it is because it requires a lot of tablegen work and will double the size of RISCVGenDAGISel.inc

Reply:
Got it!